### ISSUE TITLE
Support for BTG

### ISSUE THREAD

**Issue by morrisonbrett** [bitpay/copay-recovery#102]
Will you be adding support for Bitcoin Gold? Would love to "claim" my BTG balance pre-fork.

**Comment by Gamboster** [comment]
Hello @morrisonbrett 
The feature to claim BTG was recently added to the recovery tool..
The procedure is similar to BCH claim. You just have to select the 'btg/livenet' chain
Regards

**Comment by morrisonbrett** [comment]
Happy to say it worked. I was able to transfer the coins via the tool - But, only after I changed line 86 in recovery.service.ts from:

`      addressType: payload.addressType || "P2SH",`

to

`      "P2SH",'

You guys should consider fixing the issue that's causing this.

**Comment by Stoinkler** [comment]
This tool isn't finding my coins, are we sure it's working? It seems to import and search through the wallet ok, but there should be a balance and when it's done it says zero.

**Comment by matiu** [comment]
@Stoinkler be sure you are selecting the right chain.

**Comment by Stoinkler** [comment]
@matiu I am. Was the issue mentioned by morrisonbrett above fixed? I am trying to import a 2-of-3 multisig wallet.

**Comment by matiu** [comment]
Ill check that.later today. If this a fairly old multisig wallet? ( trying
to identify the issue)

**Comment by Stoinkler** [comment]
No, created 8/4 this year.

**Comment by Stoinkler** [comment]
Any update on this? I tried building myself with the change suggested above but don't know how to get the dependencies working to run "ng serve"

**Comment by gabrielsoltz** [comment]
Same as @Stoinkler for me. 
Could it be what @morrisonbrett says ?
Thanks in advance...

**Comment by matiu** [comment]
we will look at this issue today. thanks for the report.

**Comment by morrisonbrett** [comment]
Thanks @matiu. I see it in this PR: https://github.com/bitpay/copay-recovery/pull/109

**Comment by martyzigman** [comment]
@morrisonbrett, I don't see _recovery.service.ts_ in the this recovery tool to do your suggested fix for 

addressType: payload.addressType || "P2SH",

to

"P2SH",

**Comment by Stoinkler** [comment]
The recovery tool should work if I only enter two mnemonics of a 2-of-3 multisig correct?

**Comment by martyzigman** [comment]
I was able to get balances for my 2-of-3 multisig.  I did not need to do the addressType fix as suggested to find the balances found.

**Comment by morrisonbrett** [comment]
@martyzigman that's great.  The PR #109 isn't merged as of the time of this comment.  Hopefully it will be soon and other users that have run into this will be able to also free their coins.  There's also another PR #107 that isn't merged as of the time of this comment that increases the fee.

**Comment by martyzigman** [comment]
@morrisonbrett makes sense now.  I'll standby for the merge and when ready, I can test.

**Comment by billytsi** [comment]
Has the problem been solved? I tried to use the recovery tool for the btg and search came back zero. Thank you

**Comment by billytsi** [comment]
@matiu has the issue been solved? I did the recovery tool and it says my balance is zero btg. Thank you for help

**Comment by urbansi** [comment]
Claiming BTG from multisig fails. It finds BTG corectly, but when you want to transfer to btg address (nano leger s) it just goes on forever.

**Comment by k2a0** [comment]
How can i STOP it

**Comment by billytsi** [comment]
Terrible. Mine says zero and it's no way zero! Either post a proper link with step by step instructions or better yet do it the same way you did it with bch on our wallets.

**Comment by morenopc** [comment]
I'm getting this CORS header error trying to recover BTG (Bitcoin gold). But It worked for BTC and BCH.
```
bitpay.github.io/:1 Failed to load 
https://btgexplorer.com/api/addr/GTwHwwe7M2xqUXsExJFQqqzkeGsW4EWvko?noTxList=1: 
No 'Access-Control-Allow-Origin' header is present on the requested resource. 
Origin 'https://bitpay.github.io' is therefore not allowed access.
```
